tests/plugins/azure_storage_key_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from detect_secrets.plugins.azure_storage_key import AzureStorageKeyDetector
+
+
+class AzureStorageKeyDetector:
+
+    @pytest.mark.parametrize(
+        'payload, should_flag',
+        [
+            (
+                'AccountKey=lJzRc1YdHaAA2KCNJJ1tkYwF/+mKK6Ygw0NGe170Xu592euJv2wYUtBlV8z+qnlcNQSnIYVTkLWntUO1F8j8rQ==',
+                True,
+            ),
+        ],
+    )
+    def test_analyze(self, payload, should_flag):
+        logic = AzureStorageKeyDetector()
+        output = logic.analyze_line(payload, 1, 'mock_filename')
```

Comment:
Do you mind merging this into our `pre-v1-launch` branch instead? Unless you have a specific desire to have this on master sooner, we're current undergoing a large effort to change up the underlying architecture of this tool, and it just helps reduce merge conflicts by making changes to that branch instead.

All you would need to do is 

```
assert logic.analyze_line(filename='mock_filename', line=payload)
```

as such https://github.com/Yelp/detect-secrets/blob/pre-v1-launch/tests/plugins/slack_test.py, then point this PR to the that branch instead.

Reply:
@domanchi OK, thank for feedback. I am going to work on it.

Reply:
@domanchi updated.